cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js

```diff
@@ -142,10 +143,38 @@ function sendMessageToSlack(messageText, callback) {
   });
 }
 
+// Set the Slack channel based on the input SNS Topic to Slack Channel map
+// in the snsTopicToSlackChannel env var with format
+// {"<SNS Topic resource id>":"<slack channel name>"}
+// E.g.
+//     {"slack-low-priority-topic":"dockstore-testing",
+//     "slack-medium-priority-topic":"dockstore-dev-alerts",
+//     "slack-high-priority-topic":"dockstore-alerts",
+//     "SSM-to-Slack-SNSTopicToSlack-1VI4KZW1DSADS":"dockstore-dev-testing" }
+// input: SNS topic ARN
+function setSlackChannelBasedOnSNSTopic(topicArn) {
+  // Get the SNS Topic resource ID from the AWS ARN
+  // https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html
+  const snsTopicResourceID = topicArn.slice(topicArn.lastIndexOf(":") + 1);
+
+  for (let [topicID, channel] of Object.entries(snsTopicToSlackChannelMap)) {
```

Comment:
If it's a map (?), can't you just do `const slackChannel = snsTopicToSlackChannelMap.get(snsTopicResourceId)`?

Reply:
Its not really a map; I could not figure out how to create a map from a string without using a loop.

Reply:
Looking at [this description and example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries), it looks like just an object, so then you should be able to do `const slackChannel = snsTopicToSlackChannelMap[snsTopicResourceId]`

Reply:
fixed